app/views/rooms/show.html.erb

```diff
@@ -1,14 +1,37 @@
 <%= provide(:title, @room.listing_name) %>
+<script type='text/javascript' src='https://maps.googleapis.com/maps/api/js?key=AIzaSyANRR2kNcJ95_61PYyROVKQrRjRtqhpEUs&callback=initialize'></script>
+<script type='text/javascript'>
+ function initialize() {
```

Comment:
replace this code to this one:
`document.addEventListener("DOMContentLoaded", function(){`